src/svc.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <signal.h>
```

Comment:
This file used types that require more `#include`: `bool`, `size_t`... Please add:
```c
#include <stdbool.h>
#include <stddef.h>
```

Reply:
... I have directly done this.